### PR TITLE
SQL target keywords

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

For our Docs SEO work

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - 'MySQL'
+  - 'SQLite'
+  - 'Tables'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The **`FULL OUTER JOIN`** command combines matching rows with rows from both the left- and right-side tables.
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md:20]:
```suggestion
The **`FULL OUTER JOIN`** command combines matching rows with all rows from both the left- and right-side tables.
```

File: content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+```pseudo
+SELECT column(s)
+FROM table_A
+FULL OUTER JOIN table_B
+ON table_A.column(s) = table_B.column(s)
+WHERE condition;
+```
+
+One or more matching `column(s)` can be selected and joined from `table_A` and `table_B` based on matching columns between tables in the `ON` clause. The results can be further filtered based on a `condition` in the [`WHERE`](https://www.codecademy.com/resources/docs/sql/commands/where) clause.
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md:32]:
```suggestion
One or more matching `column(s)` can be selected and joined from `table_A` and `table_B` based on matching columns between tables in the `ON` clause to define which rows are combined. Other rows from each table appear in the result separately. The results can be further filtered based on a `condition` in the [`WHERE`](https://www.codecademy.com/resources/docs/sql/commands/where) clause.
```

File: content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Example
+
+The following example selects the all rows (`*`) from the tables `books` and `authors` that have matching author IDs less than 11:
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md:36]:
```suggestion
The following example selects the all rows (`*`) from the table `books` with an `author_id` less than 11, and any rows from the `authors` table that have a matching `id` in the filtered `books` table:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Dusch4593, First review, with only a few suggestions.

**Comment by Dusch4593** [comment]
@SSwiniarski Finished implementing your suggestions

**Review by SSwiniarski — APPROVED** [review]
Looks good for review two.

**Review by KTom101 — COMMENTED** [review]
@Dusch4593 Added a few suggestions for these entries!

File: content/sql/concepts/operators/terms/between/between.md [hunk 4]
```diff
@@ -4,22 +4,32 @@ Description: 'Selects values, inclusively of beginning and end values, within a
...
+
+## Syntax
+
+```pseudo
+SELECT *
+FROM table
+WHERE column BETWEEN value_A AND value_B;
+```
+
+The `column` must exist, and `value_A` and `value_B` must define a valid range. `BETWEEN` works with numbers, text, or date data types.
```
[the remark this answers is not in the document]
Comment (Dusch4593) [reply content/sql/concepts/operators/terms/between/between.md:28]: Nah, I think that was a typo of mine 😂

File: content/sql/concepts/commands/terms/inner-join/inner-join.md [hunk 5]
```diff
@@ -1,21 +1,40 @@
...
 CatalogContent:
   - 'learn-sql'
   - 'paths/analyze-data-with-sql'
   - 'paths/design-databases-with-postgresql'
 ---
 
-Returns a result set of all rows that have matching values in both tables, omits rows where match fails.
+The **`INNER JOIN`** command returns all rows that have matching values in both tables and omits non-matching rows.
+
+![INNER JOIN GIF](https://content.codecademy.com/courses/learn-sql/multiple-tables/inner-join.gif)
```
[the remark this answers is not in the document]
Comment (KTom101) [reply content/sql/concepts/commands/terms/inner-join/inner-join.md:22]: Yeah I was talking about width/height. It's ok then, thanks!

File: content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+The **`FULL OUTER JOIN`** command combines matching rows with all rows from both the left- and right-side tables.
+
+## Syntax
+
+```pseudo
+SELECT column(s)
+FROM table_A
+FULL OUTER JOIN table_B
+ON table_A.column(s) = table_B.column(s)
```
Comment (yangc95) [inline content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md:28]:
```suggestion
  ON table_A.column(s) = table_B.column(s)
```

File: content/sql/concepts/commands/terms/case/case.md [hunk 7]
```diff
@@ -16,9 +16,23 @@ CatalogContent:
   - 'paths/design-databases-with-postgresql'
 ---
 
-A logical test that creates different output based on the conditions of each CASE statement, closes with an END clause.
+The **`CASE`** command is a logical test that returns different output based on the conditions of each statement, closes with an `END` clause.
```
Comment (yangc95) [inline content/sql/concepts/commands/terms/case/case.md:19]:
```suggestion
The **`CASE`** command is a logical test that returns different output based on the conditions of each statement and closes with an `END` clause.
```

File: content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+WHERE condition;
+```
+
+One or more matching `column(s)` can be selected and joined from `table_A` and `table_B` based on matching columns between tables in the `ON` clause to define which rows are combined. Other rows from each table appear in the result separately. The results can be further filtered based on a `condition` in the [`WHERE`](https://www.codecademy.com/resources/docs/sql/commands/where) clause.
+
+## Example
+
+The following example selects the all rows (`*`) from the table `books` with an `author_id` less than 11, and any rows from the `authors` table that have a matching `id` in the filtered `books` table:
```
Comment (yangc95) [inline content/sql/concepts/commands/terms/full-outer-join/full-outer-join.md:36]:
```suggestion
The following example selects all the rows (`*`) from the `books` table with an `author_id` less than 11, and any rows from the `authors` table that have a matching `id` in the filtered `books` table:
```

File: content/sql/concepts/commands/terms/right-join/right-join.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+FROM table_1
+RIGHT JOIN table_2
+  ON table_1.column_name = table_2.column_name;
+```
+
+Every row in the right table is returned in the result set, and if the join condition is not met, then `NULL` values are used to fill in the columns from the left table.
+
+## Example
+
+To create a result set of every row in the `students` table combined with the `transfer_data` table where student IDs match. And if the `ON` condition is not met, then `NULL` values are used to fill in the columns from the `transfer_data` table.
```
Comment (yangc95) [inline content/sql/concepts/commands/terms/right-join/right-join.md:35]:
```suggestion
To create a result set of every row in the `students` table combined with the `transfer_data` table where student IDs match. If the `ON` condition is not met, then `NULL` values are used to fill in the columns from the `transfer_data` table.
```

Personal preference! Feel free to reject

**Review by yangc95 — COMMENTED** [review]
@Dusch4593 🙌 much improvement, took a look and only had a few comments/suggestions

**Pull request merged**